package.json

```
@@ -13,9 +13,10 @@
         "README.md"
     ],
     "scripts": {
-        "lint:docs": "markdownlint '**/*.md'",
-        "pretest": "eslint .",
-        "test": "npm run test:unit:with-coverage && npm run test:bench",
+        "lint": "run-p --continue-on-error lint:*",
+        "lint:docs": "markdownlint \"**/*.md\"",
+        "lint:js": "eslint .",
+        "test": "npm-run-all test:unit:with-coverage test:bench",
```

This this also run the tasks in parallel?

---

This one is just running these two commands in serial. We can change to parallel if you prefer.
```suggestion
        "test": "run-p test:unit:with-coverage test:bench",
```

---

Yes, this was also my concern. Thanks for clarifying.